### PR TITLE
Traitor and tweaks

### PR DESCRIPTION
**Pull request by Prakkkmak**

### CODE REVIEW

**Comment by Prakkkmak** [comment]
[Traitor](https://trello.com/c/SWX2Q7l8/41-traitor)

File: code/Player/Roles/RoleHandler.cs [hunk 1]
```diff
@@ -64,16 +64,20 @@ public void AssignRandomRole()
...
         public Role GetRandomRoleFromPreferences()
         {
-            int totalPoints = rolePreferences.Sum(rolePreference => rolesFactors[rolePreference.Value]);
-            System.Random rnd = new();
+            int totalPoints = 0;
+            foreach ((Role role, RolePreference value) in rolePreferences)
+            {
+                totalPoints += rolesFactors[value];
+            }
+            Random rnd = new();
```
Comment (Nowp) [inline code/Player/Roles/RoleHandler.cs:72]: I think we have to give a seed to a Random object

**Review by Nowp — APPROVED** [review]

**Review by Nowp — APPROVED** [review]

File: code/Player/Roles/RoleHandler.cs (hunk 1 above)
Comment (Nowp) [inline code/Player/Roles/RoleHandler.cs:72]: I think we need to give a seed to a Random object

Comment (Prakkkmak) [reply code/Player/Roles/RoleHandler.cs:72]: No need. when you don't give seed its random. 
But maybe we can make a server SEED generator and you can start game with seed :o

**Pull request merged**